app.R

```diff
@@ -8,42 +8,47 @@
 #
 
 library(shiny)
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(fpp2)
+library(ggfortify)
+library(lubridate)
 
-# Define UI for application that draws a histogram
 ui <- fluidPage(
-
-    # Application title
-    titlePanel("Old Faithful Geyser Data"),
-
-    # Sidebar with a slider input for number of bins
     sidebarLayout(
         sidebarPanel(
-            sliderInput("bins",
-                        "Number of bins:",
-                        min = 1,
-                        max = 50,
-                        value = 30)
+            selectInput(inputId = "sp", label = "Specie: ",
+                        choices = c("amar", "beth", "caca", "frni", "havi", "ilve", "libe", "saca", "vipr"),
+                        selected = "libe"),
+
         ),
 
-        # Show a plot of the generated distribution
         mainPanel(
-           plotOutput("distPlot")
+            plotOutput(outputId = "plot")
         )
     )
 )
-
-# Define server logic required to draw a histogram
+#help from  and Sophie's suggestion
 server <- function(input, output) {
 
-    output$distPlot <- renderPlot({
-        # generate bins based on input$bins from ui.R
-        x    <- faithful[, 2]
-        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+    df_subset <- reactive({
+        a <- filter(total_alive, sp == input$sp)
```

Comment:
You'll have to copy over the code from MP3 that creates `total_alive` into this `app.R` file for the app to work.

Reply:
I think total_alive is added to the app

Reply:
Okay, added the entirety of total_alive data